src/prefect/tasks/gcp/bigquery.py

```diff
@@ -403,6 +407,150 @@ def run(
         result = load_job.result()  # block until job is finished
 
 
+class BigQueryLoadFile(Task):
+    """
+    Task for insert records in a Google BigQuery table via a [load job](https://cloud.google.com/bigquery/docs/loading-data).
+    Note that all of these settings can optionally be provided or overwritten at runtime.
+
+    Args:
+        - file (Union[str, path-liike object], optional): A string or path-like object of the file to be loaded
+        - dataset_id (str, optional): the id of a destination dataset to write the
+            records to
+        - table (str, optional): the name of a destination table to write the
+            records to
+        - project (str, optional): the project to initialize the BigQuery Client with; if not provided,
+            will default to the one inferred from your credentials
+        - schema (List[bigquery.SchemaField], optional): the schema to use when creating the table
+        - location (str, optional): location of the dataset that will be queried; defaults to "US"
+        - credentials_secret (str, optional, DEPRECATED): the name of the Prefect Secret
+            containing a JSON representation of your Google Application credentials
+        - **kwargs (optional): additional kwargs to pass to the `Task` constructor
+    """
+
+    def __init__(
+        self,
+        file: Union[str, Path] = None,
+        rewind: bool = False,
+        size: int = None,
+        num_retries=_DEFAULT_NUM_RETRIES,
```

Comment:
I think this should become a kwarg and here it can default to 6:
```
num_retries: int = 6,
```

Reply:
done.